doc/index.rst

```diff
@@ -4,12 +4,10 @@ Pafy Documentation
 
 This is the documentation for pafy - a Python library to download YouTube content and retrieve metadata
 
-A quick start intro with usage examples is available in the `README <http://github.com/np1/pafy/blob/master/README.rst>`_
+A quick start intro with usage examples is available in the `README <http://github.com/mps-youtube/pafy/blob/master/README.rst>`_
```

Comment:
The default branch is longer `master`, but is `develop` now. Needs to be changed here to reflect that.